dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/scheduling/RecordingJobProgressTest.java

```diff
@@ -53,13 +56,19 @@
  *
  * @author Jan Bernitt
  */
-class RecordingJobProgressTest {
+class RecordingJobProgressTest extends TestBase {
```

Comment:
This is a pure JUnit test, for these I would also prefer we don't mix in a base class that suggests spring when it is not involved. We rather should call a static method that is directly imported.